packages/devtools_app/lib/src/config_specific/server/_server_web.dart

```diff
@@ -250,6 +250,49 @@ Future<void> resetDevToolsFile() async {
   }
 }
 
+Future<DevToolsJsonFile> requestPrimaryAppSizeFile(String path) async {
```

Comment:
these two methods are basically the same. Can we just have a single generic method that fetches files from the server. It would work for the appSize case and future cases where other files also need to be fetched from the server.

Reply:
refactored to create generic requestFile method